views/templates/hook/product.tpl

```diff
@@ -42,5 +43,8 @@
             {l s='Notify me when available' d='Modules.Emailalerts.Shop'}
         </button>
         <div class="js-mailalert-alerts"></div>
+    {else}
+        <article class="mt-1 alert alert-info" role="alert">{l s='You will be notified when available' d='Modules.Emailalerts.Shop'}</article>
```

Comment:
This looks OK to me `{l s='You will be notified when available' d='Modules.Emailalerts.Shop'}`, but when I try to translate in the back office, this string does not display for translation. Does anyone have an idea why?

Reply:
@Progi1984 @matks @NeOMakinG This looks OK to me {l s='You will be notified when available' d='Modules.Emailalerts.Shop'}, but when I try to translate in the back office, this string does not display for translation. Does anyone have an idea why?